src/controllers/users.js

```diff
@@ -1,5 +1,5 @@
 import httpStatus from 'http-status';
-import { Op } from 'sequelize';
+import sequelize, { Op } from 'sequelize';
```

Comment:
Should this be `Sequalize` with capital S? or is this somehow a shourt cut for 
```
import Sequelize from 'sequelize';
const sequelize = new Sequelize(....
.....
sequalize.where(....
```

Reply:
Right, the default import can be named however you want to name it, so `import sequelize from 'sequelize'`, `import Sequelize from 'sequelize'` or `import bubu from 'sequelize'` would all be the same. I will change it to capital case for consistency!

Reply:
Yes just wasn't sure when to use capital by convention :)